src/hooks/useKonamiCode/index.ts

```diff
@@ -8,13 +9,20 @@ import { TaskStatus } from "types/task";
 const konamiCode =
   "ArrowUpArrowUpArrowDownArrowDownArrowLeftArrowRightArrowLeftArrowRightba";
 
-const useKonamiCode = (onActivate: () => void) => {
+const useKonamiCode = () => {
   const [pressedKeys, setPressedKeys] = useState<string[]>([]);
   const dispatchToast = useToastContext();
   const { sendEvent } = useAprilFoolsAnalytics();
-  const downHandler = ({ key }: KeyboardEvent) => {
+  const downHandler = ({ key, target }: KeyboardEvent) => {
+    // Ignore key presses if the user is typing in an input
+    if (target instanceof HTMLInputElement) return;
```

Comment:
If you want to cover more bases you could add `HTMLTextAreaElement`, `HTMLButtonElement`, and `HTMLSelectElement`